src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -405,6 +405,27 @@ This command prepares an OpenShift installer-provisioned infrastructure (IPI) on
 | `--natt-port` `<int>`            | IPsec NAT Traversal port (default 4500)
 | `--vxlan-port` `<int>`           | Internal VXLAN port (default 4800)
 
+#### `prepare rhos`
+
+`subctl cloud prepare rhos [flags]`
+
+This command prepares an OpenShift installer-provisioned infrastructure (IPI) on OpenStack cloud for Submariner installation.
+
+| Flag                             | Description
+|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| `--cloud-entry` `<string>`       | the specific cloud configuration to use from the clouds.yaml (default `openstack`)
+| `--dedicated-gateway`            | Whether a dedicated gateway node has to be deployed (default true)
+| `--gateway-instance` `<string>`  | Type of gateway instance machine (default `PnTAE.CPU_4_Memory_8192_Disk_50`)
+| `--gateways` `<int>`             | Number of dedicated gateways to deploy (default 1)
+| `--infra-id` `<string>`          | Openstack infra ID
```

Comment:
```suggestion
| `--infra-id` `<string>`          | OpenStack infra ID
```

There are a bunch of other instances of "Openstack->OpenStack".

Reply:
Done.